b2share/config.py

```diff
@@ -47,6 +41,180 @@
     account_json_loader, account_json_patch_loader,
 )
 
+from invenio_search.config import SEARCH_ELASTIC_HOSTS
+
+SEARCH_ELASTIC_HOSTS = []
+
+for elastic_host in os.environ.get("ELASTIC_HOSTS", "localhost:9200").split(','):
+    try:
+        (host, port) = elastic_host.split(':')
+    except:
+        host = elastic_host
+        port = 9200
+
+    SEARCH_ELASTIC_HOSTS.append(dict(host=host, port=port, use_ssl=False, ssl_show_warn=False))
+
+def _(x):
+    """Identity function used to trigger string extraction."""
+    return x
+
+# Rate limiting
+# =============
+#: Storage for ratelimiter.
+RATELIMIT_STORAGE_URL = os.environ.get("INVENIO_RATELIMIT_STORAGE_URL", 'redis://localhost:6379/3')
+
+# I18N
+# ====
+#: Default language
+BABEL_DEFAULT_LANGUAGE = 'en'
+#: Default time zone
+BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
+#: Other supported languages (do not include the default language in list).
+I18N_LANGUAGES = [
+    # ('fr', _('French'))
+]
+
+# Base templates
+# ==============
+#: Global base template.
+BASE_TEMPLATE = 'b2share_main/base.html'
+#: Cover page base template (used for e.g. login/sign-up).
+COVER_TEMPLATE = 'invenio_theme/page_cover.html'
+#: Footer base template.
+MAIN_TEMPLATE = 'invenio_theme/footer.html'
+#: Header base template.
+HEADER_TEMPLATE = 'invenio_theme/header.html'
+#: Settings base template.
+SETTINGS_TEMPLATE = 'invenio_theme/page_settings.html'
+
+# Theme configuration
+# ===================
+#: Site name
+THEME_SITENAME = _('B2SHARE')
+#: Use default frontpage.
+THEME_FRONTPAGE = True
+#: Frontpage title.
+THEME_FRONTPAGE_TITLE = _('B2SHARE')
+#: Frontpage template.
+THEME_FRONTPAGE_TEMPLATE = 'b2share_main/page.html'
```

Comment:
This is essentially the B2SHARE V2 webui/app/index.html.
**!! Build of webui and packaging with B2SHARE backend code needs to be adopted.**
New Dockerfile addresses this.